Triplehelix490/sketch_jul12a_fade/sketch_jul12a_fade.ino

```diff
@@ -1,20 +1,22 @@
+//    ||||||||||
 //    \/\/\/\/\/change these factors to change the characteristics of the components.
 int pin = 9;
 int brightness = 0;
 int fadeAmount = 5;
 int timeOn = 400;
 
+
 //setup code: this pulls the program from the Arduino pre-program
-void setup() {
+void setup(){
   pinMode(pin, OUTPUT);
 }
 
 //main code: the main code which produce the desired effects for the components
-void loop (){
+void loop(){
       analogWrite(pin, brightness);
-        brightness = brightness + fadeAmount
-        if ( brightness <= 0 ||brightness >= 255) {
-        fadeAmount = - fadeAmount;
-}
-delay(timeOn);
-}
+        brightness = brightness + fadeAmount;
+             if ( brightness <= 0||brightness >= 255){
+                 fadeAmount = - fadeAmount;
+              }
+       delay(timeOn);
+}       
```

Comment:
Don't be too hard on yourself on how you have structured this, keep doing what your doing then you will crack this in no time Adam.#dontgiveupAdammmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmm#